### ISSUE TITLE
WARNING: Unknown module: jide.common specified to --add-exports

### ISSUE THREAD

**Issue by kamraanchal** [igvteam/igv#1611]
`Running **igv version 2.18.4 with JAVA** gives the below warnings`  on a CentOS linux system

]$ igv.sh --help
echo Using bundled JDK.
openjdk version "17.0.9" 2023-10-17
OpenJDK Runtime Environment Temurin-17.0.9+9 (build 17.0.9+9)
OpenJDK 64-Bit Server VM Temurin-17.0.9+9 (build 17.0.9+9, mixed mode, sharing)
WARNING: Unknown module: jide.common specified to --add-exports
WARNING: Unknown module: jide.common specified to --add-exports
WARNING: Unknown module: jide.common specified to --add-exports
WARNING: Unknown module: jide.common specified to --add-exports
WARNING: Unknown module: jide.common specified to --add-exports
WARNING: package com.sun.java.swing.plaf.windows not in java.desktop
WARNING: package sun.awt.windows not in java.desktop
WARNING: Unknown module: jide.common specified to --add-exports
WARNING: Unknown module: jide.common specified to --add-exports

Can someone suggest how to get rid of these warnings ?

**Comment by jrobinso** [comment]
Hi, I don't think you can't get rid of those warnings.

**Comment by kamraanchal** [comment]
@jrobinso How can i get rid for these warnings . Any suggestions ?

**Comment by jrobinso** [comment]
@kamraanchal I've already replied, did you miss it?  No, I don't have any suggestion, AFAIK you can't get rid of them.  What issue are they causing?

**Comment by jrobinso** [comment]
I see, sorry about that.   Please tell your customers they can ignore these warnings.   They will have no impact no IGV's functionality.

**Comment by kamraanchal** [comment]
@jrobinso Thank you for the information. We will communicate the same.

**Comment by fthommen** [comment]
@jrobinso 🤔 When I install a tool according to its installation instructions and use it according to its usage instructions and it starts right away with nine warnings, then I'd see this either as a (although lesser) bug in the software, the installation instructions or the usage instructions.
Also it would be interesting to know from the developer side, what is the reason for these warnings and why we can't get rid of them
(_I am also involved in the maintenance of the affected software installation mentioned by the OP_)

**Comment by jrobinso** [comment]
@fthommen fair enough, consider it a lesser bug then.  I will reopen it but can't give any ETAs at this time.
